l10n_mapper_generator/lib/l10n_mapper_builder.dart

```diff
@@ -14,13 +14,15 @@ Builder l10nMapperBuilder(BuilderOptions options) {
   final locale = options.config['locale'] as bool? ?? true;
   final parseL10n = options.config['parseL10n'] as bool? ?? true;
   final message = options.config['message'] as String?;
+  final mapperWhitelist = (options.config['mapper-whitelist'] as List<dynamic>?)?.map<String>((item) => item.toString()).toList() ?? [];
```

Comment:
I'm not skipping them. I'm skippibg all others.

Reply:
Been quite busy to look at this for some time now but the point is; the term `white-list` contradicts its applicable usecase here as this means `free-access` to some resource. 

Since the intension here is to implement the inverse (i.e avoid generating access to this provided keys), adopting the term `black-list` concept will better suit since its inline with proposed intensions.

This i why I proposed `key-exceptions` as this inherently applies the same concept as `blacklist`.

Reply:
It is just not feasible for me to add black-list for 300+ translation keys, just to be able to include 30 and bypass the others. This is my suggestion. If you don't like it, just ignore and close the PR. I have this repository forked with the changes I need.